clamav-rest/bootstrap.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/bin/sh
 set -m
 
 host=${HOST:-127.0.0.1}
 port=${PORT:-3310}
-# I *think* this is miliseconds. The default is 500.
+filesize=${MAXSIZE:-10240MB}
 timeout=${TIMEOUT:-15000}
 
-echo "using clamd server: $host:$port timeout: $timeout"
+echo "using clamd server: $host:$port, timeout: $timeout, max file size: $filesize"
 
 # start in background
-java -jar /var/clamav-rest/clamav-rest-1.0.2.jar --clamd.host=$host --clamd.port=$port --clamd.timeout=$timeout
+java -jar /var/clamav-rest/clamav-rest-1.0.2.jar --clamd.host=$host --clamd.port=$port --clamd.maxfilesize=$filesize --clamd.maxrequestsize=$filesize
```

Comment:
Removing `--clamd.timeout`? Is it not needed anymore?

Reply:
Good spot, I missed this